README.md

```diff
@@ -304,6 +306,20 @@ earnings.symbol_id # 4563
 See [#earnings](https://iextrading.com/developer/docs/#earnings) for detailed documentation or [earnings.rb](lib/iex/resources/earnings.rb) for returned fields.
 
 
+### Get Sector Performance
+
+Fetches earnings for a symbol.
```

Comment:
Missed this one. It should be something like: `Fetches latest sector's performance.`